src/redisearch_api.c

```diff
@@ -27,12 +28,13 @@ int RediSearch_GetCApiVersion() {
   return REDISEARCH_CAPI_VERSION;
 }
 
-IndexSpec* RediSearch_CreateIndex(const char* name, const RSIndexOptions* options) {
+RefManager* RediSearch_CreateIndex(const char* name, const RSIndexOptions* options) {
```

Comment:
Maybe add a general comment that we return the `RefManager*` so we will not have the break to LLAPI but wherever we get it we know that it is actually a `StrongRef`

Reply:
Thought: Since `RSIndex` is `RefManager`, consider using `RSIndex` as the type to process in the functions of this file